x-pack/plugins/painless_lab/public/application/index.tsx

```diff
@@ -23,7 +23,8 @@ interface AppDependencies {
 
 export function renderApp(
   element: HTMLElement | null,
-  { http, I18nContext, uiSettings, links }: AppDependencies
+  { http, I18nContext, uiSettings, links }: AppDependencies,
+  props: MainProps
```

Comment:
I would suggest keeping `Main` a propless component as far as possible, if we have the ChromeStart service here we can just add it to our app context to bring this in line with the current pattern?